src/installers/helm.ts

```diff
@@ -195,18 +195,26 @@ error: E_COMMAND_FAILED`)
 
     let multiUserFlag = ''
     let tlsFlag = ''
-    let setOptions = ''
+    let setOptions = []
 
     if (flags.multiuser) {
       multiUserFlag = `-f ${destDir}values/multi-user.yaml`
     }
 
     if (flags.tls) {
-      setOptions = `--set global.cheDomain=${flags.domain} --set global.tls.email='${ctx.tlsEmail}'`
+      setOptions.push(`--set global.cheDomain=${flags.domain} --set global.tls.email='${ctx.tlsEmail}'`)
       tlsFlag = `-f ${destDir}values/tls.yaml`
     }
 
-    let command = `helm upgrade --install che --force --namespace ${flags.chenamespace} --set global.ingressDomain=${flags.domain} ${setOptions} --set cheImage=${flags.cheimage} --set global.cheWorkspacesNamespace=${flags.chenamespace} --set che.workspace.devfileRegistryUrl=${flags['devfile-registry-url']} --set che.workspace.pluginRegistryUrl=${flags['plugin-registry-url']} ${multiUserFlag} ${tlsFlag} ${destDir}`
+    if (flags['plugin-registry-url']) {
```

Comment:
what is happening if you use server:start without `--plugin-registry-url` ? it is failing no ?

Reply:
it doesn't set helm parameter so value from helm chart is used. By default registries will be deployed in your minikube and URLs will be set to those https://github.com/eclipse/che/pull/13890

Reply:
ok it was not clear enough for me that this PR can only be merged after https://github.com/eclipse/che/pull/13890

Reply:
well, I don't think that this must be merged after. Helm charts currently does not set the registry urls, but then `che.properties` values are used, which are again set to `openshift.io` ones https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L580
So basically nothing happens when we merge this one. The chectl registries url params was added just 2 days ago https://github.com/che-incubator/chectl/pull/189